docs/modules/general/main.mdx

```diff
@@ -167,6 +167,18 @@ The maps version should follow the versioning schema `major.minor.patch`.
         </td>
         <td />
       </tr>
+      <tr>
+        <td>
+          <label>{`<created>`}</label>
+        </td>
+        <td>
+          The date on which this map was initially released.
+        </td>
+        <td>
+          <span className="badge badge--primary">String</span>
```

Comment:
It can't just be any string, it has to be a specific string using the ISO date format (yyyy-mm-dd).